packages/lcio/package.py

```diff
@@ -27,7 +27,7 @@ class Lcio(CMakePackage):
             description='Use the specified C++ standard when building.')
     variant("jar", default=False,
     description="Turn on to build/install lcio.jar")
-    variant("rootdict", default=True,
```

Comment:
Why this change? I think we agreed at one point that `+rootdict` is a good default.

Reply:
This one wasn't compiling for me before with that, but I just uninstalled and compiled, and its working now. I'll revert it.